uber/automated_emails.py

```diff
@@ -23,53 +25,211 @@ def __init__(self, model, subject, template, filter, *, sender=None, extra_data=
         self.sender = sender or c.REGDESK_EMAIL
         self.instances[self.subject] = self
         self.ident = (ident or self.subject).format(EVENT_NAME=c.EVENT_NAME)
+        self.when = when
+
+        # after each daemon run, this will be set to the number of emails that would have been sent out but weren't
+        # because they were not marked as approved.  Useful as a metric of how many emails need human intervention in
+        # order to be approved for sending.
+        #
+        # A value of -1 means we haven't run yet, or email sending is disabled. Always check for > -1.
+        self.unapproved_emails_not_sent = -1
```

Comment:
Can we use None here instead of -1? I hate magic integers. It is probably an irrational hatred, but checking for `is None` still seems more intuitive than `== -1`.